tools/scitokens-admin-oauthcerts

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
Use `/usr/bin/python`.  When `python setup.py install` is performed, it will rewrite this line as appropriate.